### PR TITLE
Adding Hugo metadata to Release ReadME

### PR DESCRIPTION
**Pull request by vishakhanihore**
Addition of metadata for Hugo as per conversation on [slack](https://kubernetes.slack.com/archives/C2C40FMNF/p1580302816258200)

### CODE REVIEW

File: releases/release-1.18/README.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
+---
+title: "Kubernetes Release 1.18"
```
Comment (justaugustus) [inline releases/release-1.18/README.md]:
```suggestion
title: "Kubernetes 1.18 Release"
```

File: releases/release-1.18/README.md [hunk 2]
```diff
@@ -1,3 +1,9 @@
...
+weight: 98
+slug: "kubernetes-release"
```
Comment (justaugustus) [inline releases/release-1.18/README.md]:
```suggestion
slug: "kubernetes-release-118" 
```

File: releases/release-1.18/README.md [hunk 3]
```diff
@@ -1,3 +1,9 @@
+---
+title: "Kubernetes Release 1.18"
+weight: 98
```
Comment (justaugustus) [inline releases/release-1.18/README.md]: I'm not intimately familiar with the weights.
Was there a reason for choosing 98?

Comment (vishakhanihore) [reply releases/release-1.18/README.md]: Yup, We have several other side menu tabs that need to be shown below the Release Tab. Hence going with weight 98.

File: releases/release-1.18/README.md (hunk 2 above)
Comment (vishakhanihore) [reply releases/release-1.18/README.md]: Done

File: releases/release-1.18/README.md (hunk 3 above)
Comment (justaugustus) [reply releases/release-1.18/README.md]: Thanks!

File: releases/release-1.18/README.md [hunk 4]
```diff
@@ -1,3 +1,9 @@
+---
+title: "Kubernetes 1.18 Release"
+weight: 98
+slug: "kubernetes-118-release"
```
Comment (mrbobbytables) [inline releases/release-1.18/README.md]:
```suggestion
slug: "release" 
```

The slug for this should just be `release` or `current-release`. The filename will be changed during the import to match.

Comment (vishakhanihore) [reply releases/release-1.18/README.md]: Making the change 👍

Comment (vishakhanihore) [reply releases/release-1.18/README.md]: Done !

**Comment by vishakhanihore** [comment]
@justaugustus lgtm will be needed again , I just squashed my commits 😅

**Comment by onlydole** [comment]
Wow - woke up and saw the assignment, but y'all are FAST. Thanks for making it easy 😄 Happy Wednesday!

**Pull request merged**